File: VideoClub/src/main/java/com/videoClub/service/AgeClassifierService.java
package com.videoClub.service;

import java.util.List;

import com.videoClub.model.AgeClassifier;

public interface AgeClassifierService {

	public List<AgeClassifier> getAll();
	public List<AgeClassifier> saveAll(List<AgeClassifier> categories);
}

File: VideoClubFront/src/app/services/age-classifier.service.ts
import {Injectable} from '@angular/core';
import {HttpClient, HttpHeaders} from '@angular/common/http';
import {Observable} from 'rxjs';
import { map } from "rxjs/operators";
import { AgeClassifier } from '../model/age-classifier.model';

@Injectable({providedIn: 'root'})
export class AgeClassifierService {

  constructor(private http: HttpClient) {
    
  }

    getAll = (): Observable<AgeClassifier[]> => {
      return this.http.get<AgeClassifier[]>("/api/age_classifiers").pipe(
        map( (res: any) => {
            return res;
        })  );
    }
}